src/shared/apis/queryCache.ts

```diff
@@ -124,19 +125,29 @@ class QueryCache {
       isCustomTime,
       mutex: RunQueryPromiseMutex<RunQueryResult>(),
       status: RemoteDataState.Loading,
+      paused: false,
     }
     return this.cache[queryID]
   }
 
   resetCacheByID = (id: string): void => {
-    if (!this.cache[id]) {
+    if (!this.cache[id] || this.cache[id]?.paused) {
       return
     }
     delete this.cache[id]
   }
 
   resetCache = (): void => {
-    this.cache = {}
+    for (const key of Object.keys(this.cache)) {
+      if (!this.cache[key].paused) {
+        delete this.cache[key]
+      }
+    }
+    // this.cache = {}
```

Comment:
what is this

Reply:
something that should and will no longer exist, thanks for the catch